jetson/nav/rover.cpp

```diff
@@ -112,6 +77,51 @@ bool Rover::drive(double distance, double bearing, double threshold, double dt)
     return false;
 }// drive()
 
+bool Rover::driveBackwards(const Odometry& destination, double stopDistance, double dt) {
+    double distance = estimateDistance(mOdometry, destination);
+    double bearing = estimateBearing(mOdometry, destination);
+    return driveBackwards(distance, bearing, stopDistance, dt);
+}// drive()
+
+bool Rover::driveBackwards(double distance, double bearing, double threshold, double dt) {
```

Comment:
This also seems mostly like a copy/pasta of the regular drive. Let's reduce duplication here too